mesh/profiles/views.py

```diff
@@ -1,6 +1,10 @@
+from django.core.exceptions import ObjectDoesNotExist
 from django.views.decorators.csrf import csrf_exempt
 from django.http import JsonResponse
 
+from mesh.profiles.models import Profile
+
+
 @csrf_exempt
```

Comment:
We shouldn't be using @csrf_exempt tags in our code. It's a security risk if it gets into production. 

I should have disabled the CSRF middleware in the backend while we work on a proper CSRF implementation, eliminating the need for this tag. Have you tried not using it?